src/fem_bc.md

```diff
@@ -216,11 +216,11 @@ a.AddBoundaryIntegrator(new MassIntegrator(rbcACoef), rbc_marker);
 
 In the Discontinuous Galerkin (DG) formulation the
 [Natural](#natural-boundary-conditions),
-[Neumann](#neumann-boundary-conditions), and
-[Robin](#robin-boundary-conditions) can be implemented in precisely
-the same manner as in the continuous case.  However, since DG basis
-functions have no degrees of freedom associated with the boundary,
-Dirichlet boundary conditions must be handled differently.
+[Neumann](#neumann-boundary-conditions), and [Robin](#robin-boundary-conditions)
+can be implemented in a similar the same manner as in the continuous case (using
+the appropriate _boundary face_ integrators).  However, since DG basis functions
+have no degrees of freedom associated with the boundary, Dirichlet boundary
```

Comment:
```suggestion
can be implemented in a similar the same manner as in the continuous case
(adding the appropriate `LinearFormIntegrator` as a _boundary face integrator_
instead of _boundary integrator_).  However, since DG basis functions
have no degrees of freedom associated with the boundary, Dirichlet boundary
```
This suggestion fleshes out what using the appropriate integrator entails, but I suppose the particular word choice is subjective.